Build failure: rustdesk

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

build it

### Can Hydra reproduce this build failure?

Hydra doesn’t try to build the package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
: ToolExecError: Command LC_ALL="C" "/nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-I" "/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/common" "-I" "/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/externals/nv-codec-headers_n12.1.14.0/include/ffnvcodec" "-I" "/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/common/platform/linux" "-I" "/homeless-shelter/installed/x64-linux/include" "-Wall" "-Wextra" "-fno-omit-frame-pointer" "-o" "/build/source/target/x86_64-unknown-linux-gnu/release/build/hwcodec-91d4e87aa117d7ee/out/d6032c8462032270-ffmpeg_ram_decode.o" "-c" "/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/ffmpeg_ram/ffmpeg_ram_decode.cpp" with args cc did not execute successfully (status code exit status: 1).
rustdesk> error: failed to run custom build command for `hwcodec v0.7.1 (https://github.com/rustdesk-org/hwcodec#17c1dbb3)`
rustdesk>
rustdesk> Caused by:
rustdesk>   process didn't exit successfully: `/build/source/target/release/build/hwcodec-5522301457fb2f78/build-script-build` (exit status: 1)
rustdesk>   --- stdout
rustdesk>   cargo:rerun-if-changed=src
rustdesk>   cargo:rerun-if-changed=deps
rustdesk>   cargo:rerun-if-changed=/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/externals
rustdesk>   cargo:rerun-if-changed=/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp
rustdesk>   cargo:info=x64-linux
rustdesk>   cargo:rustc-link-search=native=/homeless-shelter/installed/x64-linux/lib
rustdesk>   cargo:rustc-link-lib=avcodec
rustdesk>   cargo:rustc-link-lib=avutil
rustdesk>   cargo:rustc-link-lib=avformat
rustdesk>   cargo:include=/homeless-shelter/installed/x64-linux/include
rustdesk>   cargo:rustc-link-lib=drm
rustdesk>   cargo:rustc-link-lib=X11
rustdesk>   cargo:rustc-link-lib=stdc++
rustdesk>   cargo:rustc-link-lib=z
rustdesk>   OUT_DIR = Some(/build/source/target/x86_64-unknown-linux-gnu/release/build/hwcodec-91d4e87aa117d7ee/out)
rustdesk>   OPT_LEVEL = Some(3)
rustdesk>   TARGET = Some(x86_64-unknown-linux-gnu)
rustdesk>   HOST = Some(x86_64-unknown-linux-gnu)
rustdesk>   cargo:rerun-if-env-changed=CC_x86_64-unknown-linux-gnu
rustdesk>   CC_x86_64-unknown-linux-gnu = None
rustdesk>   cargo:rerun-if-env-changed=CC_x86_64_unknown_linux_gnu
rustdesk>   CC_x86_64_unknown_linux_gnu = None
rustdesk>   cargo:rerun-if-env-changed=HOST_CC
rustdesk>   HOST_CC = Some(/nix/store/ag7fc58qzff4j1sgd7v98irhcr6vcg8v-gcc-wrapper-14.3.0/bin/cc)
rustdesk>   RUSTC_WRAPPER = None
rustdesk>   cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
rustdesk>   cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
rustdesk>   CRATE_CC_NO_DEFAULTS = None
rustdesk>   DEBUG = Some(false)
rustdesk>   CARGO_CFG_TARGET_FEATURE = Some(fxsr,sse,sse2)
rustdesk>   cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
rustdesk>   CFLAGS_x86_64-unknown-linux-gnu = None
rustdesk>   cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
rustdesk>   CFLAGS_x86_64_unknown_linux_gnu = None
rustdesk>   cargo:rerun-if-env-changed=HOST_CFLAGS
rustdesk>   HOST_CFLAGS = None
rustdesk>   cargo:rerun-if-env-changed=CFLAGS
rustdesk>   CFLAGS = None
rustdesk>   CARGO_ENCODED_RUSTFLAGS = Some(-Cforce-frame-pointers=yes)
rustdesk>   OUT_DIR = Some(/build/source/target/x86_64-unknown-linux-gnu/release/build/hwcodec-91d4e87aa117d7ee/out)
rustdesk>   cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
rustdesk>   cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
rustdesk>   CRATE_CC_NO_DEFAULTS = None
rustdesk>   TARGET = Some(x86_64-unknown-linux-gnu)
rustdesk>   CARGO_CFG_TARGET_FEATURE = Some(fxsr,sse,sse2)
rustdesk>   HOST = Some(x86_64-unknown-linux-gnu)
rustdesk>   cargo:rerun-if-env-changed=CFLAGS_x86_64-unknown-linux-gnu
rustdesk>   CFLAGS_x86_64-unknown-linux-gnu = None
rustdesk>   cargo:rerun-if-env-changed=CFLAGS_x86_64_unknown_linux_gnu
rustdesk>   CFLAGS_x86_64_unknown_linux_gnu = None
rustdesk>   cargo:rerun-if-env-changed=HOST_CFLAGS
rustdesk>   HOST_CFLAGS = None
rustdesk>   cargo:rerun-if-env-changed=CFLAGS
rustdesk>   CFLAGS = None
rustdesk>   cargo:warning=/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/common/util.cpp: In function 'void util_encode::set_av_codec_ctx(AVCodecContext*, const std::string&, int, int, int)':
rustdesk>   cargo:warning=/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/common/util.cpp:59:18: error: 'FF_PROFILE_H264_HIGH' was not declared in this scope; did you mean 'AV_PROFILE_H264_HIGH'?
rustdesk>   cargo:warning=   59 |     c->profile = FF_PROFILE_H264_HIGH;
rustdesk>   cargo:warning=      |                  ^~~~~~~~~~~~~~~~~~~~
rustdesk>   cargo:warning=      |                  AV_PROFILE_H264_HIGH
rustdesk>   cargo:warning=/build/rustdesk-1.4.3-vendor/hwcodec-0.7.1/cpp/common/util.cpp:61:18: error: 'FF_PROFILE_HEVC_MAIN' was not declared in this scope; did you mean 'AV_PROFILE_HEVC_MAIN'?
rustdesk>   cargo:warning=   61 |     c->profile = FF_PROFILE_HEVC_MAIN;
rustdesk>   cargo:warning=      |                  ^~~~~~~~~~~~~~~~~~~~
rustdesk>   cargo:warning=      |                  AV_PROFILE_HEVC_MAIN
rustdesk>   exit status: 0
rustdesk>   exit status: 0
rustdesk>   exit status: 1
```

### Additional context

_No response_

### System metadata

- system: `"x86_64-linux"`
 - host os: `Linux 6.17.7-cachyos, NixOS, 25.11 (Xantusia), 25.11.20251111.9da7f1c`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.94.0
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/qweered/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/qweered/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/qweered/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/qweered/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/yqsdfnfk19s82cf3dg2j3a2ys9x8rfm1-lix-2.94.0-pre-20251018_6e2edbff930d/share`
 - nixpkgs: `/nix/store/cjjpqrp9xy44j5fjh3mygyxy3i7mslj2-source`

### Notify maintainers

@ocfox  @Leixb 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ru/rustdesk-flutter/package.nix
+++ b/pkgs/by-name/ru/rustdesk-flutter/package.nix
@@ -5,7 +5,7 @@
   copyDesktopItems,
   fetchFromGitHub,
   flutter329,
-  ffmpeg,
+  ffmpeg_7,
   gst_all_1,
   fuse3,
   libXtst,
@@ -119,7 +119,7 @@ flutter329.buildFlutterApplication rec {
   ];
 
   buildInputs = [
-    ffmpeg
+    ffmpeg_7
     fuse3
     gst_all_1.gst-plugins-base
     gst_all_1.gstreamer

